packages/core/test/text/textTests.tsx

```diff
@@ -12,11 +12,19 @@ import * as React from "react";
 import { Classes, Text } from "../../src/index";
 
 describe("<Text>", () => {
-    it("ellipsizes contents", () => {
-        const textContent = "text";
+    it("renders jsx text children", () => {
```

Comment:
wait wait i meant _another_ test. like "it truncates a string". "it truncates jsx content".